aep/general/0164/aep.md.j2

```diff
@@ -60,20 +60,26 @@ A resource that supports soft delete **should** provide an `Undelete` method:
 
 {% endtabs %}
 
-### Long-running undelete
+### Create
 
-Some resources take longer to undelete a resource than is reasonable for a
-regular API request. In this situation, the API **should** follow the
-long-running request pattern AEP-151.
+Create methods **must** adhere to one of the following:
+
+- If a user attempts a create on a soft-deleted resource, the create **must**
+  succeed, acting as if the resource did not exist previously.
+- The create must accept a field (query parameter for OAS),
```

Comment:
Can you put must in bold? (That's also how the site generator knows to change the color)